site/docs/get-started/quickstart.md

```diff
@@ -70,10 +70,10 @@ helm install cert-manager jetstack/cert-manager \
 		cd fybrik
 		helm dependency update charts/vault
 		helm install vault charts/vault --create-namespace -n fybrik-system \
-			--set "vault.global.openshift=true" \
 			--set "vault.injector.enabled=false" \
 			--set "vault.server.dev.enabled=true" \
-			--values charts/vault/env/dev/vault-single-cluster-values.yaml
+			--values charts/vault/env/dev/vault-single-cluster-values.yaml \
```

Comment:
Its better to place `vault-openshift-values.yaml` under `charts/vault/` dir as it's not related to the vault mode (dev/ha)

Reply:
make sense, thanks, will do it.